src/server.cc

```diff
@@ -63,6 +63,10 @@ int server(data d, config const& c, std::string_view const motis_version) {
   auto r = runner{server_config.n_threads_, 1024U};
   auto qr = net::query_router{net::fiber_exec{ioc, r.ch_}};
   qr.add_header("Server", std::format("MOTIS {}", motis_version));
+  if (c.server_->data_attribution_link_) {
+    qr.add_header("Link", std::format("<{}>; rel=\"license\"",
```

Comment:
```suggestion
    qr.add_header("Link", fmt::format("<{}>; rel=\"license\"",
```

`#include <format>` is missing. But we're usually only using `fmt::` because it gets continuous improvements (unlike the `std::` version). So for consistency please switch to `fmt`.